addons/mail/static/tests/helpers/mock_server.js

```diff
@@ -105,6 +105,9 @@ MockServer.include({
             const context = args.context;
             return this._mockRouteMailChatPost(uuid, message_content, context);
         }
+        if (route === '/mail/get_suggested_recipients') {
+            return this._mockGetSuggestedRecipient(args);
```

Comment:
Prefer destructuring the args here to keep the signature of the method itself as close as possible to the one from the python. (see the above route as example if needed)